Hello World Project PR

nurbolsultanov opened the pull request:
my first push request on GitHub (Hello World Project)
What have I done on this Project:
1. Created ReadMe file for the project.
2. Added Linters to the project.
3. Added .gitignore file.

nurbolsultanov committed:
Create .gitignore

added .gitignore file with node_modules

nurbolsultanov committed:
Create linters.yml

added linters for HTML & CSS

nurbolsultanov committed:
Update README.md

edited readme file

nurbolsultanov committed:
Create index.html

created html file

nurbolsultanov committed:
Create style.css

created css file

nurbolsultanov committed:
Update .gitignore

added test.md file to .gitignore

nurbolsultanov committed:
Create testlogo.png.png

added logo

nurbolsultanov committed:
logo

renamed logo file

nurbolsultanov committed:
edited linter

trying to fix linters

IngDavidTM commented on README.md:
```diff
@@ -1,2 +1,29 @@
+  
+**Nurbol Sultanov**
+
+- GitHub: [@everinurmind](https://github.com/everinurmind)
```
- [x] You did a good job implementing the `README` file :+1: but kindly use [Microverse Readme template](https://github.com/microverseinc/readme-template) and make sure all the main sections are present in yours like key features, Future Features, Contributions, and  Getting Started that describe how a new developer could make use of your project by following some steps. It is important because it would make your project looks more professional and this is one of the first things that hiring managers or your potential colleagues will check when considering you for a position on their team. To learn more you could read [Document your repo in a professional way](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/clean_repo.md)

IngDavidTM replied:
- [x] Kindly don't forget to implement a [MIT license](https://choosealicense.com/licenses/mit/) because software license tells others what they can and can't do with your source code, so it's essential to make an informed decision. You can also set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). :+1:

IngDavidTM commented on style.css:
```diff
@@ -0,0 +1,7 @@
+}
+
+body {
+    background-color: beige;
```
- [x] As a [project requirement](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/hello_microverse_project.md) you should be sure that there are [no linter errors](https://github.com/microverseinc/linters-config) :+1: . I recommend you check the indentation of lines 2 and 6, and also left an empty line after all the elements :+1: .

IngDavidTM reviewed (changes requested):
## Hi @everinurmind  🙋‍♂️, 

Good job so far!
You have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

<img src="https://media2.giphy.com/media/31Q36EHJ7C5ZoIMw79/giphy.gif"/>

## Top highlight 🎯
- The Pull Request follows the GitHub Flow ✅ 

## Suggested changes
- [x] Kindly don't forget that Pull Request should have a descriptive title telling you what changes are inside, if you have doubts I recommend you to check [Professional repo - basic rules](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md) :+1: 

- [x] As a general requirement you should set up linters in your local env. Kindly don't forget to add `Webhint`, `Stylelint`, etc, to your project. [Here](https://github.com/microverseinc/linters-config/tree/master/html-css) you can find more information about the linters configuration.
- [x] _Check the comments under the review._ ⬇

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

nurbolsultanov committed:
Update style.css

nurbolsultanov committed:
Create LICENSE.md

added License to my project

nurbolsultanov committed:
Update README.md

added license and changed some parts

nurbolsultanov committed:
Update style.css

checked the indentation of lines 2 and 6, and also left an empty line after all the elements

nurbolsultanov committed:
Create LICENSE.md

nurbolsultanov committed:
delete

deleted

nurbolsultanov committed:
Update style.css

nurbolsultanov committed:
Update style.css

nurbolsultanov committed:
Update README.md

edited readme file with more detail of the project

nurbolsultanov committed:
added linter files

nurbolsultanov committed:
added new packages

nurbolsultanov committed:
readme

fixed readme to link to LICENSE.MD

nurbolsultanov committed:
license fixed

nurbolsultanov committed:
last fix

removed .

DuaneDave reviewed (changes requested):
Hi @everinurmind 👋, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
✔️ Quick implementation of requested changes 💯 
✔️ Professional documentation of project

 ## Required Changes ♻️
- According to the previous review, the PR summary must be descriptive in order to make it easy for others to know what changes have been made in this project. (HINT: you can enumerate the list of tasks done here.)

Click on the three dots on the right and select edit to change the PR summary.


![1FCC9150-FC33-4363-9BE5-57B8B5E70778](https://user-images.githubusercontent.com/84930355/211871621-005973bf-1b58-432c-99b2-f4c64f306c15.jpeg)



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

ridwanediallo reviewed (approved):
# STATUS: APPROVED 🟢
## Hi @everinurmind , 
![Congratulations!GIF](https://user-images.githubusercontent.com/55918904/191301390-43d62dfa-bfdc-49c5-943b-4d72f4ea5bb8.gif)

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- GitHub flow is used correctly. ✔
- All linters checks are ok. ✔

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
